src/components/features/ProductSearch/ProductSearch.js

```diff
@@ -12,6 +12,13 @@ const ProductSearch = () => (
       <FontAwesomeIcon className={styles.icon} icon={faListUl} />
       <select name='' id=''>
         <option value=''>Select a category</option>
+        <option value=''>Home</option>
+        <option value=''>Furniture</option>
+        <option value=''>Chair</option>
+        <option value=''>Table</option>
+        <option value=''>Sofa</option>
+        <option value=''>Bedroom</option>
+        <option value=''>Blog</option>
```

Comment:
Dobrze, że dodałaś wszystkie kategorie, ale w zadaniu chodzi przede wszystkim o zmianę struktury, tego menu aby pozbyć się elementu `<select>` bo w tej chwili użytkownik musi kliknąć w menu aby zobaczyć kategorię a chodzi o to, że wystarczy najechać myszką na menu w celu jego rozwinięcia.

W opisie zadania jest propzycja do zamiany elementu `<select>` na strukturę `ul>li` . W ramach przypomnienia proponuje zajrzeć do modułu 2 gdzie analogiczne menu było budowane.